From a3640775bb4416932e41e9804a08e46a1ff29712 Mon Sep 17 00:00:00 2001
From: Aaron Madlon-Kay <aaron@madlon-kay.com>
Date: Wed, 18 Aug 2021 08:49:33 +0900
Subject: [PATCH] Allow supplying additional CFLAGS

Package managers may want to supply default CFLAGS that should be used *in addition* to project-specific ones.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1c465fe..365ddf4 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ ifneq ($(strip $(COV_TEST)),)
 endif
 
 CC ?= gcc
-CFLAGS ?=-Wall -DLIBPCRE -g $(CFLAGS_COV)
+CFLAGS +=-Wall -DLIBPCRE -g $(CFLAGS_COV)
 
 LIBS=-lm -lpcre2-8
 OBJS=sslh-conf.o common.o sslh-main.o probe.o tls.o argtable3.o udp-listener.o collection.o gap.o
